docker/resources/envs/.env.local

```diff
@@ -1,11 +1,11 @@
 # RUST_LOG=debug
 
-ZKV_NODE_KEY_FILE="/data/config/node_key.dat"
```

Comment:
To answer your question, the node key is randomly generated, just like any other secret.
Maybe we should also add `ZKV_SECRET_PHRASE` in these templates (possibly with some predefined value like `"YOUR_SECRET_PHRASE"`), to make it clear where it should be provided?

Reply:
I will add this to each `.env.local` file where ZKV_NODE_KEY is present with empty "" value to make sure if someone forgets to change COMMENT value entrypoint script will not fail since it is processing these vars now